internal/compile/compile.go

```diff
@@ -266,8 +266,10 @@ var stackEffect = [...]int8{
 	SLICE:       -3,
 	STAR:        -1,
 	TRUE:        +1,
+	UMINUS:      0,
```

Comment:
Just to confirm, this is just for readability, right? It's not strictly necessary to set zero values here?

Reply:
Yes, just for completeness.